src/pubhub/pub_hub.bal

```diff
@@ -113,6 +136,15 @@ function startHubAndRegisterTopic() returns websub:WebSubHub {
                                 retryConfig: {
                                     count:  3,
                                     intervalInMillis: 5000
+                                },
+                                followRedirects: { enabled: true, maxCount: 5 },
+                                timeoutInMillis: 5*60000, // Check
+                                auth: { authHandler: outboundBasicAuthHandler },
+                                secureSocket: {
+                                    trustStore: {
+                                        path: config:getAsString("truststore"),
```

Comment:
Again do we need this for the hub?

Reply:
So I tried adding the secure socket config for the subscriber service. With that added, the intent verification request seems to fail from the hub to the subscriber.

Shall we clarify the behaviour?